src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -485,7 +485,7 @@ enable_unity_manila_backend=false
 # registry.connect.redhat.com
 # By default, this version of the image of the RHOSP release is pulled.
 # Change to a different version if a specific image is required.
-manila_unity_container_version=16.1-1
+manila_unity_container_version=16.1-2
```

Comment:
Why 16.1-2 out of curiosity?

Reply:
> Why 16.1-2 out of curiosity?

It is not Z2, it is tag on the container